src/styles/output.css

```
@@ -6963,6 +6974,38 @@
+  top: 16px;
+  right: 16px;
+  z-index: 50;
+  max-width: 420px;
+  padding: 14px 16px;
+  border: 1px solid rgba(47, 212, 131, 0.32);
+  border-radius: 12px;
+  background: color-mix(in srgb, #2fd483 14%, #16161a);
+  @supports (color: color-mix(in lab, red, red)) {
+    background: color-mix(in srgb, var(--color-success) 14%, var(--color-surface));
+  }
+  color: var(--color-foreground);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)